app/frontend/shared/Zxcvbn.res

```diff
@@ -0,0 +1,65 @@
+exception UnexpectedZxcvbnScore(int)
+
+type zxcvbnFeedback = {suggestions: array<string>}
+type zxcvbnResponse = {
+  score: int,
+  feedback: zxcvbnFeedback,
+}
+
+@module("zxcvbn") external zxcvbn: (string, array<string>) => zxcvbnResponse = "default"
+let ts = I18n.t(~scope="components.Zxcvbn")
+
+type strength = Weak | Fair | Medium | Strong
+type t = {
+  score: int,
+  strength: strength,
```

Comment:
@kaisersakhi I'm not a huge fan of the way `score: int` is being kept in the type, even though the `strength` variant is the same information. I think that `score` can be removed from here, but I also see how the stored integer value makes it easy to determine color in the `colorClass` function, so I suppose it's _okay_ to keep it here.

Reply:
@harigopal While coding it, I had this in my mind that I should avoid using int but I couldn't find a easier way to do it!